game/src/main/java/net/demilich/metastone/game/entities/weapons/Weapon.java

```diff
@@ -94,4 +89,9 @@ public String toString() {
 		result += "]";
 		return result;
 	}
+
+	public void setDurability(int durability) {
```

Comment:
Ah, didn't mean to add that. Realized it was unnecessary since it's an Actor. I'll fix it later.